src/health_misinfo_shared/fine_tuning.py

```diff
@@ -465,6 +469,38 @@ def infer_claims(video_id: str, transcript: list[dict]) -> Iterable[dict[str, An
     return all_responses
 
 
+def infer_multimodal_claims(
+    bucket_name: str, video_path: str, model_name: str = "gemini-1.5-flash-001"
+) -> Iterable[dict[str, Any]]:
+    vertexai.init(project=GCP_PROJECT_ID, location=GCP_LLM_LOCATION)
+    model = GenerativeModel(model_name=model_name)
+    response = model.generate_content(
+        [
+            Part.from_uri("gs://" + video_path, mime_type="video/mp4"),
+            MULTIMODAL_RAPHAEL_PROMPT,
+        ],
+        safety_settings=VIDEO_SAFETY_SETTINGS,
+        generation_config={
+            "candidate_count": 1,
+            "max_output_tokens": 8192,
+            "temperature": 0,
+            "top_p": 1,
+        },
+    )
+
+    if not len(response.candidates):
+        raise Exception(f"No model output: possible reason: {response.prompt_feedback}")
+
+    candidate = response.candidates[0]
+    text = candidate.text
+    if text.startswith("```"):
+        text = text.strip("`")
+    if text.startswith("json"):
+        text = text[4:]
+    print(text)
```

Comment:
Can probably remove this now, or only print it if `json.loads` throws an exception.